src/commands/test.ts

```diff
@@ -0,0 +1,134 @@
+import {Command} from './command';
+import * as wct from 'web-component-tester';
+// import * as process from 'process';
+
+export class TestCommand implements Command {
+  name = 'test';
+
+  description = 'Runs web-component-tester';
+
+  args = [
```

Comment:
Yeah, this is somewhat temporary, and obviously not very maintainable. I either want to standardize on command-line-args style declarations, or add some mapping functions.